hpcbench/toolbox/contextlib_ext.py

```diff
@@ -102,3 +102,35 @@ def mkdtemp(*args, **kwargs):
     finally:
         if remove:
             shutil.rmtree(path)
+
+
+@contextlib.contextmanager
+def modified_environ(*remove, **update):
```

Comment:
Question: can you not use this cool utility now in driver for the `pre_execute` and `post_execute` ?

Reply:
Yes. absolutely.